src/hal/sensor/sensor.h

```diff
@@ -0,0 +1,17 @@
+
+#ifndef __SENSOR_HAL_H__
+#define __SENSOR_HAL_H__
+
+#include <stdint.h>
+#include "../../hal/hal.h"
+
+// Initializes the pressure and airflow sensors
+int16_t sensorHalInit(void);
+
+// Gets the airflow sensor value
+int16_t airflowSensorHalGetValue(int16_t *value);
```

Comment:
Return codes can be int; values need definite size, return codes are ok as `int` (unless I later want to make them their own type....)

Reply:
Done